document/js-api/index.bs

```diff
@@ -1383,10 +1388,14 @@ The <dfn attribute for="Exception">stack</dfn> getter steps are:
 <h4 id="js-exceptions">JavaScript exceptions</h4>
 
 The <dfn>JavaScript exception tag</dfn> is a [=tag address=] reserved by this
-specification to distinguish exceptions originating from JavaScript.
+specification to distinguish exceptions originating from JavaScript. It is exposed
+to JavaScript code via {{WebAssembly}}.{{JSTag}} a global pre-initailized (TODO: what's ths spec-y way to say this?)
+Tag object [=create a Tag object|created=] from the JavaScript exception tag address.
+
 
 For any [=associated store=] |store|, the result of
-[=tag_parameters=](|store|, [=JavaScript exception tag=]) must be « ».
+[=tag_parameters=](|store|, [=JavaScript exception tag=]) must be « [=externref=] ».
+
```

Comment:
I think what we should be doing (as the "spec-y" thing, and in order to preerve abstraction boundaries) is to invoke `tag_alloc` from the core spec's embedding interface to create that tag, with the respective tag type. That could happen lazily, i.e., the JS spec does the allocation the first time the tag address is needed and then memoizes it.

Btw, I think the embedding spec should also provide a `tag_type` function, analogous to the other `*_type` functions.